code/game/objects/items/reagent_containers/pill.dm

```diff
@@ -241,3 +241,24 @@
 	//pill_desc = "An ultrazine pill. A highly-potent, long-lasting combination CNS and muscle stimulant. Extremely addictive."
 	list_reagents = list(/datum/reagent/medicine/ultrazine = 5)
 	pill_id = 21
+
+// packet pills, normal sized but carry different amounts.
+
+/obj/item/reagent_containers/pill/tricordrazine/packet
+	pill_desc = "A larger than usual tricordrazine pill. Broad spectrum medication that slowly heals all damage types."
+	list_reagents = list(/datum/reagent/medicine/tricordrazine = 20)
```

Comment:
I don't understand why these have 20u when it was established that 10u give toxloss. Clarify both the reasons behind 20u and whether 10u give toxloss.

Reply:
Alright, it was explained in Discord by MJP

> .25
> every tick
> you'll heal that off naturally unless you're premeedding on bica 24/7
> it's to stop people from doing 14/14/14 or whatever